frontend/packages/kubevirt-plugin/integration-tests/tests/utils/consts.ts

```diff
@@ -39,6 +39,8 @@ export const V2V_VM_IMPORT_TIMEOUT = 3600 * SEC;
 // Wizard strings
 export const IMPORT_WIZARD_CONN_TO_NEW_INSTANCE = 'Connect to New Instance';
 export const NOT_RECOMMENDED_BUS_TYPE_WARN = 'Not recommended bus type';
+// Some times we need to use existing VMWare instance, which name always starts from 'administrator'
+export const IMPORT_WIZARD_CONN_TO_EXISTING_INSTANCE = 'administrator';
```

Comment:
I would maybe suggest more descriptive name for this one, `IMPORT_WIZARD_CONN_NAME_PREFIX`